recipes/orange3-timeseries/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: orange3-timeseries
+  version: "0.2.5"
+
+source:
+  fn: Orange3-Timeseries-0.2.5.tar.gz
+  url: https://pypi.python.org/packages/ed/7d/d6b6a2b3ec051eb5a10f5e99b9459a51572f1c4bf3aacd72ba2e9a5a0d77/Orange3-Timeseries-0.2.5.tar.gz
+  md5: 09f3d1c95674683aa53d98df9b539bf5
+  patches:
+    - skip-symlinking-datasets.patch
+
+build:
+  preserve_egg_dir: True
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - statsmodels >=0.6.1
+    - orange3
```

Comment:
Without statsmodels, the recipe wouldn't build locally. I added Orange for the symlink hack in setup.py which assumes the dir is there, before I patched it all away. So it's probably a stale dependency I'll remove.